### PR TITLE
Add Ghost User-Agent

### LINKED ISSUE #378: New User-Agent not detected (PR to follow)
Hi,

I noticed that the Ghost publishing platform webhook User-Agent is not detected / parsed.

Examples:
```
Ghost/2.13.1+moya (https://github.com/TryGhost/Ghost)
Ghost/2.13.1 (https://github.com/TryGhost/Ghost)
```

PR and tests will follow.

Thanks,

Tim Birkett.

### PR DESCRIPTION
**Pull request by pysysops**
This PR closes #378 and allows for parsing / detection of the User-Agent used by Ghost to make webhook calls to services.

Any feedback welcome!

Tim.

### CODE REVIEW

File: tests/test_ua.yaml [hunk 1]
```diff
@@ -7733,7 +7733,7 @@ test_cases:
     family: 'Salesforce'
     major: '15'
     minor: '2'
-    patch:
```
Comment (kingo55) [inline tests/test_ua.yaml]: Should these formatting changes to other tests be added in a separate PR?

Comment (commenthol) [reply tests/test_ua.yaml]: These are just whitespace changes. We should invest in a editorconfig first to prevent such cases.
So I would not mind.

**Comment by commenthol** [comment]
@pysysops Could you please rebase?

**Comment by commenthol** [comment]
Thanks
